docs/06-concepts/11-authentication/05-custom-overrides.md

```diff
@@ -42,6 +46,39 @@ List<String> scopes = extractScopes(token);
 Set<Scope> userScopes = scopes.map((scope) => Scope(scope)).toSet();
 ```
 
+### Handling revoked authentication 
+
+When a user's authentication is revoked, the server must be notified to respect the changes (e.g. to close method streams). Invoke the `session.messages.authenticationRevoked` method and raise the appropriate event to notify the server.
+
+```dart
+var userId = 1;
+var removedScopes = ['write'];
+var message = RevokedAuthenticationScope(
+  scopes: removedScopes,
```

Comment:
@SandPod As you said yourself, maybe rename this to `revokedScopes` to be completely consistent with language 🎉

Reply:
Good catch! 🎉